build/create-icons.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
```

Comment:
Do we actually have to do this at the postinstall? Can't this be a regular component like `<tify-icon name="test">`?

Reply:
It cannot, unless we want all 6000+ icons to be compiled into the TIFY package; tree shaking does not help. Besides, using icon components like this is exactly what the creators of Vue are doing.

Reply:
Hm not even like [this](https://stackoverflow.com/a/61534450)?

Reply:
Nope.